pgmpy/models/FactorGraph.py

```diff
@@ -391,3 +391,39 @@ def get_partition_function(self):
             raise ValueError('DiscreteFactor for all the random variables not defined.')
 
         return np.sum(factor.values)
+
+    def copy(self):
+        """
+        Returns a copy of the model.
+
+        Returns
+        -------
+        FactorGraph : Copy of FactorGraph
+
+        Examples
+        --------
+        >>> from pgmpy.models import FactorGraph
```

Comment:
@ankurankan Sorry , but can you please tell me what's wrong here as i have observed many similar import statements and cannot identify what is wrong.